cpu/sam0_common/periph/rtc.c

```diff
@@ -65,11 +65,8 @@ static inline void _rtc_set_enabled(bool on)
 #ifdef CPU_SAMD21
 static void _rtc_clock_setup(void)
 {
-    /* Setup clock GCLK2 with OSC32K divided by 32 */
-    GCLK->GENDIV.reg = GCLK_GENDIV_ID(2) | GCLK_GENDIV_DIV(4);
-    GCLK->GENCTRL.bit.DIVSEL = 1;
-    GCLK->CLKCTRL.reg = GCLK_CLKCTRL_CLKEN | GCLK_CLKCTRL_GEN(2) | GCLK_CLKCTRL_ID_RTC;
-    while (GCLK->STATUS.bit.SYNCBUSY) {}
```

Comment:
Any good reason to drop the SYNCBUSY ?

Reply:
Since the GCLK is not reconfigured anymore, it should not be needed.
I can check with the data sheet.

Reply:
IIRC, the SYNCBUSY is here to ensure GCLK has been enabled and is now running.

Reply:
I suggest we leave the SYNCBUSY test in. If it is not needed, then it can be done in a separate PR together with the other cases (rtt.c, gpio.c, pwm.c)

Reply:
It's back now.